interview-questions.md

```diff
@@ -8,36 +8,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: I would fix this by adding a cohort_id column tot he student model using rails g migration add_columns_to_students in the terminal. Then I would add the column to the migration in the app using add_column and defing the model, name, and datatype. Then run migrations using rails db:migrate updating the schema. The foreign key is attached to the students model because the cohort has_many students and the students belong_to cohort, so each individual student will be attached to a cohorts id with the new cohort_id column.
 
-Researched answer:
+Researched answer: I would fix this by adding a cohort_id column tot he student model using rails g migration add_columns_to_students in the terminal. Then I would add the column to the migration in the app using add_column and defing the model, name, and datatype. Then run migrations using rails db:migrate updating the schema. The foreign key is attached to the students model because the cohort has_many students and the students belong_to cohort, so each individual student will be attached to a cohorts id with the new cohort_id column. A foreign key is a field that refers to a primary key on another table.
```

Comment:
Your initial answer is great!  I did see you added a little more in the researched answer, but I would always recommend digging a little further rather than copying your initial answer.